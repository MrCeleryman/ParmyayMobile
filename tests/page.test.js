import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import Page from "../src/home";

describe("app", () => {
	it("renders correctly", () => {
		const tree = renderer.create(
			<Page />
		);
	});

	it("Snapshot matches", () => {
		expect(renderer.create(
			<Page />
		)).toMatchSnapshot();
	});
});
