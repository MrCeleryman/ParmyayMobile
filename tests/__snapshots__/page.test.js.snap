// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app Snapshot matches 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
  style={
    Object {
      "backgroundColor": "#FFFCF9",
      "flex": 1,
    }
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 20,
          "margin": 10,
          "textAlign": "center",
        }
      }
    >
      Welcome to Parmyaymobile!
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-start",
        }
      }
    >
      <Image
        source={
          Object {
            "uri": "http://pizzalinos.letseat.at/images/social_urbanspoon.png?1445209169",
          }
        }
        style={
          Object {
            "height": 50,
            "width": 50,
          }
        }
      />
      <TextInput
        autoCorrect={false}
        onChangeText={undefined}
        placeholder="Email"
        secureTextEntry={undefined}
        style={
          Object {
            "color": "#000",
            "flex": 3,
            "flexDirection": "row",
            "fontSize": 16,
            "lineHeight": 21,
            "paddingLeft": 5,
            "paddingRight": 5,
          }
        }
        value={undefined}
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-start",
        }
      }
    >
      <Image
        source={
          Object {
            "uri": "https://cdn4.iconfinder.com/data/icons/standard-free-icons/139/Safety01-128.png",
          }
        }
        style={
          Object {
            "height": 50,
            "width": 50,
          }
        }
      />
      <TextInput
        autoCorrect={false}
        onChangeText={undefined}
        placeholder="Password"
        secureTextEntry={undefined}
        style={
          Object {
            "color": "#000",
            "flex": 3,
            "flexDirection": "row",
            "fontSize": 16,
            "lineHeight": 21,
            "paddingLeft": 5,
            "paddingRight": 5,
          }
        }
        value={undefined}
      />
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "margin": 10,
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#8FA3AD",
            "borderRadius": 2,
            "elevation": 0,
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "margin": 5,
            "opacity": 1,
            "padding": 15,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#FFFCF9",
                "fontWeight": "500",
              },
              undefined,
              null,
            ]
          }
        >
          SIGNUP
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#8FA3AD",
            "borderRadius": 2,
            "elevation": 0,
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "margin": 5,
            "opacity": 1,
            "padding": 15,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#FFFCF9",
                "fontWeight": "500",
              },
              undefined,
              null,
            ]
          }
        >
          SIGNIN
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#5D34AF",
          "borderRadius": 2,
          "elevation": 5,
          "flexDirection": "row",
          "justifyContent": "center",
          "margin": 5,
          "opacity": 1,
          "padding": 15,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#FFFCF9",
              "fontWeight": "500",
            },
            undefined,
            null,
          ]
        }
      >
        GO TO TEST PAGE
      </Text>
    </View>
    <View
      style={
        Object {
          "transform": Array [
            Object {
              "rotate": "0deg",
            },
          ],
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#5D34AF",
            "borderRadius": 2,
            "elevation": 5,
            "flexDirection": "row",
            "justifyContent": "center",
            "margin": 5,
            "opacity": 1,
            "padding": 15,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#FFFCF9",
                "fontWeight": "500",
              },
              undefined,
              null,
            ]
          }
        >
          SPIN ME!
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "flex-start",
          "marginHorizontal": 20,
        }
      }
    >
      <View
        style={
          Object {
            "transform": Array [
              Object {
                "rotate": "0deg",
              },
            ],
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 30,
              "width": 30,
            }
          }
        />
      </View>
      <TextInput
        onChangeText={[Function]}
        placeholder="Type here"
        style={
          Object {
            "flex": 1,
          }
        }
      />
    </View>
  </View>
</RCTScrollView>
`;
